src/styles/utils/mixin.scss

```diff
@@ -0,0 +1,23 @@
+@mixin number-day {
+  @for $cell from 1 through 31 {
+    & :nth-child(#{$cell})::before {
+      content: '#{$cell}';
+    }
+  }
+}
+
+@mixin weak-day-lentgh {
```

Comment:
It seems there is a typo in the mixin name 'weak-day-lentgh'. Did you mean 'week-day-length'? It's important for mixin names to be clear and correctly spelled to maintain code readability and prevent confusion.